<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>faceDetectionTestClass</class>
 <widget class="QWidget" name="faceDetectionTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>faceDetectionTest</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="10,4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3,2">
     <item>
      <widget class="QLabel" name="videoLb">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 127);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="srcPicLb">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>130</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 255, 255);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>检测阈值：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="similarLb">
           <property name="text">
            <string>相似度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="choseSrcBtn">
           <property name="text">
            <string>选择元图片</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="4,10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startCamareBtn">
           <property name="text">
            <string>start</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveCamarePicBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>save</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>视频模式</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>正常模式</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">videoModeBtnGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="faceDetectCheck">
            <property name="text">
             <string>人脸检测</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">videoModeBtnGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="face2faceCheck">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>人脸匹配</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">videoModeBtnGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="faceDetectionTest.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="videoModeBtnGroup"/>
 </buttongroups>
</ui>
